tests/conftest.py

```diff
@@ -206,8 +210,76 @@ def expected_catalog_from_db():
          ]}
     ]})
 
+@pytest.fixture
```

Comment:
Since we removed the fixture because of issues it was causing, we needed to create a fixture version of this one for one of the tests.